File: bee_crowd/problems_dart/bee_1080.dart
import 'dart:io';

import '../../io_utils.dart';

void main() {
  //entrada
    var valor;
    var counter = 1;
    var maior = - double.maxFinite;
    var posicao_maior = counter;
    
    //process

    while (counter <= 10) {
        valor = int.parse(stdin.readLineSync()?? "");
        
        counter++;

        if (valor > maior) {
            maior = valor;
            posicao_maior = counter;
        }
        

    }

    //saida
    print(maior);
    print(posicao_maior);
}

File: bee_crowd/problems_dart/bee_1080.js
import { input } from "../../io_utils.js";
function main() {

    //entrada
    let valor
    let counter = 1
    let maior = Number.MIN_VALUE
    let posicao_maior = counter
    
    //process

    while (counter <= 10) {
        valor = Number(input(`Digite um Valor ${counter}: `))
        
        counter++

        if (valor > maior) {
            maior = valor
            posicao_maior = counter
        }
        

    }

    //saida

    console.log(`${maior}`)
    
    console.log(`${posicao_maior}`)


}
main()